homeassistant/components/rflink/sensor.py

```diff
@@ -102,6 +220,32 @@
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
     ),
+    SensorEntityDescription(
+        key="winddirection",
+        name="Wind direction",
+        icon="mdi:compass",
+    ),
```

Comment:
Also:
```python
        state_class=SensorStateClass.MEASUREMENT,
```

Reply:
Done + added unit of measurement = DEGREE